projects/core/directives/hint/hint-position.directive.ts

```diff
@@ -18,6 +18,7 @@ const TOP = 0;
 const LEFT = 1;
 
 @Directive({
+    standalone: true,
     selector: '[tuiHint]:not(ng-container):not(ng-template)',
```

Comment:
We probably need to move it to `hostDirectives` of `TuiHintDirective` and make `:is(never)` selector so it does not add noise for autoimports.

Reply:
Can I use a separate PR?